### DISCUSSION TITLE
SMS

### DISCUSSION THREAD

**Discussion by CommanderKeen4** [egzumer/uv-k5-firmware-custom#501] [Q&A]
Can you make one version of your firmware without the analyzer and that it has the sending of short messages? And optional DSB TX/RX?

**Comment by kamilsss655** [comment]
And if you could include an app to track calories that'd be great ;)

**Comment by egzumer** [comment]
I don't fulfill special requests. You have all the instructions in the readme, you may choose whatever is available and build it yourself.

**Comment by CommanderKeen4** [comment]
I totally understand, it's normal. If you are going to develop something and it will not be a special job for you, this is how this proposal was intended... This is more an idea that would be for the needs of rescue and protection that radio clubs would have in this way to help in the event of accidents and disasters, I would rather used a more useful station configuration than compared to cosmetic additions such as an analyzer...
